extra_data/tests/test_streamer.py

```diff
@@ -59,6 +61,18 @@ def server(protocol_version):
         yield s
 
 
+@pytest.fixture(scope='function')
+def file_server(mock_fxe_raw_run):
+    port = 3333
+    args = shlex.split(
+        f'karabo-bridge-serve-files {mock_fxe_raw_run} {port} '
+        f'--append-detector-modules'
+    )
```

Comment:
```suggestion
    args = [
        'karabo-bridge-serve-files', str(mock_fxe_raw_run), str(port),
        '--append-detector-modules'
    ]
```

Grumble grumble, don't format things into a single string just to split them up again. E.g. if `mock_fxe_raw_run` contains a space, then formatting it into a command string like this turns it into two arguments.